draft-ietf-scitt-architecture.md

```diff
@@ -93,7 +100,9 @@ The rise in popularity of verifiable data structures as a mechanism to make acto
 
 This document defines a generic, interoperable and scalable architecture to enable transparency across any supply chain with minimum adoption barriers.
 It provides flexibility, enabling interoperability across different implementations of Transparency Services with various auditing and compliance requirements.
-Producers can register their Signed Statements on any Transparency Service, with the guarantee that all Consumers will be able to verify them.
+Issuers can register their Signed Statements on any Transparency Service, with the guarantee that all Consumers will be able to verify them.
+
+Within the SCITT Architecture, a producer is known as an Issuer, and a Consumer is known as a Verifier.
```

Comment:
Oh, good catch @aj-stein-nist 
Since both producer and consumer are NOT terms, they should both be lower, while Issuer and Verifier are named terms, and should be upper. 
Committing your suggestion for catching this, then fixing in the PR.